command/ca/init.go

```diff
@@ -845,3 +849,13 @@ func normalize(name string) string {
 	}
 	return name
 }
+
+// allEmpty loops through all strings in the slice and returns if
+// all are empty (length 0).
+func allEmpty(parts []string) bool {
+	sumLength := 0
+	for _, p := range parts {
+		sumLength += len(p)
```

Comment:
This is fine, although seems like you could just stop iterating when you find one that is `> 0`, no? I imagine this list will be pretty short so probably doesn't make much difference.